### ISSUE TITLE
hglib.min.js

### ISSUE THREAD

**Issue by astrovsky01** [hms-dbmi/higlass-app#16]
Is there any way to get access to the source code for hglib.min.js? I need to modify the method of export for files to not use Download, and any modifications I'm doing on the file that's being pulled on startup causes the interface to not be displayed at all. Thank you!

**Comment by pkerpedjiev-zymergen** [comment]
Yup, that's the higlass javascript component. You can find the source code here: https://github.com/higlass/higlass/

You can build that using :

```
npm install
npm run build
```
If you want to include that in the Docker container, you'll have to modify the following line in the Dockerfile template to use your locally build higlass instead of a published one:

https://github.com/higlass/higlass-docker/blob/4001ee86f653de77a822eacd4922ae8bbb8b8253/web-context/Dockerfile.template#L85

If you get it working and think that your change might be useful to others, please do submit a PR to the higlass repo.
